classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @student << student
+  end
```

Comment:
- [ ] In the `add_student` method when adding a student to a classroom, also set the classroom for the student. Also in line 11, there should be `@students`.

Hint:
```ruby
def add_student
 @students << student
 student.classroom = self
end
```